lib/graphql/schema/object.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module GraphQL
+  class Schema
+    class Object < GraphQL::Schema::Member
```

Comment:
In a world where we'll all be writing a _lot_ of `class BlahType < GraphQL::Schema::Object` I can imagine wishing that {Object, Scalar, Interface, Union, Enum, InputObject} were not namespaced to Schema, just to save some characters.

I understand not wanting to have an explosion of classes at the `GraphQL` level, but I think its merited for those 6 to live there (as is the case in the `GraphQL::ObjectType` syntax)

Perhaps, for the sake of internal consistency, they could be aliased to these shorthands?
```ruby
module GraphQL
  Object = Schema::Object
  ...
```

Reply:
Although, if you are encouraging the pattern
```ruby
class GraphQL
  class BaseObject < Schema::Object
```
then perhaps this is unnecessary

Reply:
@bessey Arel has some shortcuts to do this, for example, you can use `Arel.sql` instead of `Arel::Nodes::SqlLiteral`. Maybe a similar pattern can be used for GraphQL schema definitions as well?

Reply:
I think the intention is to mirror what Rails has recently done with `ApplicationRecord` for example. So the docs would need to encourage that behaviour. If that's the case, the Rails generator could even generate the `BaseObject` (or whatever it's called).

Reply:
Yeah, that's what I'm going for, maybe I can make that clear by starting _each_ section with an example like that.